.github/ISSUE_TEMPLATE/add_triage_owner_rotation.yml

```diff
@@ -0,0 +1,52 @@
+name: Configure Triage Owner Rotation
+description: Add new triage owner rotation configuration
+title: "[Round Robin]: <title>"
+labels: ["enhancement"]
+assignees:
+  - suhaibmujahid
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this form!
+
+        **Note:** information filled in this form will be publicly viewable.
+  - type: input
+    id: team_name
+    attributes:
+      label: Your Team Name
+      description: It must exactly match the team name on Bugzilla.
+      placeholder: e.g., CI and Quality Tools
+    validations:
+      required: true
+  - type: dropdown
+    id: coverage
+    attributes:
+      label: Rotation Coverage
+      options:
+        - All team components
+        - Only specific components
+    validations:
```

Comment:
let's add a description, pointing to the docs about the team to components mapping (not everybody is aware of it): https://wiki.mozilla.org/BMO/UserGuide#Team_to_components_mapping.